src/univariate_finite/arrays.jl

```diff
@@ -181,6 +164,27 @@ function Base.Broadcast.broadcasted(
     return broadcast(pdf, u, cat)
 end
 
+# logpdf.(u, cv)
+# logpdf.(u, v)
+# logpdf.(u, raw)
+for typ in [:CategoricalValue, :(AbstractArray{<:CategoricalValue{V,R},N}), :(Union{V,AbstractArray{V,N}})]
+ eval(quote function Base.Broadcast.broadcasted(
+         ::typeof(logpdf),
+         u::UniFinArr{S,V,R,P,N},
+         cv::$typ) where {S,V,R,P,N}
+
+        # Start with the pdf
+        result = pdf.(u,cv)
+
+        # Take the long of each entry in-place
```

Comment:
Typo from my original comment :)

```suggestion
        # Take the log of each entry in-place
```